Code/GraphMol/RascalMCES/RascalMCES.cpp

```diff
@@ -255,14 +255,26 @@ void getBondLabels(const ROMol &mol, const RascalOptions &opts,
   getAtomLabels(mol, opts, atomLabels);
   bondLabels = std::vector<std::string>(mol.getNumBonds());
   for (const auto &b : mol.bonds()) {
-    if (b->getBeginAtom()->getAtomicNum() < b->getEndAtom()->getAtomicNum()) {
-      bondLabels[b->getIdx()] = atomLabels[b->getBeginAtomIdx()] +
-                                std::to_string(b->getBondType()) +
-                                atomLabels[b->getEndAtomIdx()];
+    if (b->getBeginAtom()->getAtomicNum() == b->getEndAtom()->getAtomicNum()) {
```

Comment:
Rather selfishly, perhaps, I went with just what I needed to get the job done.  Element and bond type are already encoded, so I don't think getTotalDegree would be any different from getDegree.  Charge might be useful, maybe, and you could make a case for options for things like treating all halogens the same.  Given that enhancements now go in in the bug fixing releases, I think it would be ok to work on an on-demand basis.